tests/test_main.py

```diff
@@ -22,87 +32,115 @@ def created_person_list(people_data):
+    assert person_inst.age == age, "Person instance should have attribute 'age'"
 
 
 def test_create_person_list_all_persons(people_data, created_person_list):
-
     assert all(isinstance(person, Person) for person in created_person_list), (
         "All elements in result of 'create_person_list' should be instance "
         "of Person class"
     )
-    assert len(created_person_list) == len(people_data), (
-        "Length of initial list should equal to length of function result"
-    )
+    assert len(created_person_list) == len(
+        people_data
+    ), "Length of initial list should equal to length of function result"
 
 
 def test_create_person_list_order(people_data, created_person_list):
-   assert [person_dict['name'] for person_dict in people_data] == [
-       person.name for person in created_person_list
-   ], (
-        "Order in function result should be the same"
-    )
+    assert [person_dict["name"] for person_dict in people_data] == [
+        person.name for person in created_person_list
+    ], "Order in function result should be the same"
 
 
 def test_create_person_list_has_wife(people_data, created_person_list):
-    assert hasattr(created_person_list[0], 'wife'), (
+    assert hasattr(created_person_list[0], "wife"), (
         f"Person with 'name' {created_person_list[0].name} should have "
         f"attribute 'wife' with name {people_data[0].wife.name}"
     )
 
 
-def test_create_person_list_has_wife_and_wife_have_husband(people_data, created_person_list):
-    assert hasattr(created_person_list[0], 'wife') and \
-           created_person_list[0].wife.husband == created_person_list[0], (
+def test_create_person_list_has_wife_and_wife_have_husband(
+    people_data, created_person_list
+):
+    assert (
+        hasattr(created_person_list[0], "wife")
+        and created_person_list[0].wife.husband == created_person_list[0]
+    ), (
         f"Person with 'name' {created_person_list[0].name} should have "
         f"attribute 'wife' with name {created_person_list[0].wife.name} and "
         f"Person.wife.husband should links to that Person"
     )
 
 
 def test_create_person_list_has_no_wife(people_data, created_person_list):
-    assert hasattr(created_person_list[1], 'wife') is False, (
+    assert hasattr(created_person_list[1], "wife") is False, (
         f"Person with 'name' {created_person_list[1].name} should not have "
         f"attribute wife"
     )
 
 
 def test_person_class_attribute_people(people_data, created_person_list):
-    assert all(isinstance(person, Person) for person in Person.people.values()), (
-        "All elements of Person class attribute 'people' should be Person instances"
-    )
-    assert len(Person.people) == len(people_data), (
-        "Length of Person class attribute people should be equal to length of initial list"
+    assert all(
+        isinstance(person, Person) for person in Person.people.values()
```

Comment:
The test assumes `Person.people` is a dictionary with `Person` instances as values. Ensure that the `Person` class is implemented to support this structure.